x-pack/plugins/infra/kibana.json

```diff
@@ -15,8 +15,7 @@
     "triggersActionsUi",
     "observability",
     "ruleRegistry",
-    "unifiedSearch",
-    "lens"
```

Comment:
we might want to leave this for now as we are thinking to use Lens Embeddables for other things